### PR TITLE
[test/plugin]immigrate pulsar scenario

### PR DESCRIPTION
**Pull request by codelipenghui**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues
#3583

### CODE REVIEW

**Comment by wu-sheng** [comment]
Your codes may miss one Apache 2.0 header. Rat check fails.

File: .github/workflows/plugins-test.yaml [hunk 1]
```diff
@@ -492,3 +492,28 @@ jobs:
         run: ./mvnw -f test/plugin/pom.xml clean package -DskipTests docker:build -DBUILD_NO=local >/dev/null
       - name: Run mysql 5.1.2-8.0.15 (53)
         run: bash test/plugin/run.sh mysql-scenario
+
+  Pulsar:
```
Comment (wu-sheng) [inline .github/workflows/plugins-test.yaml]: Pulsar test is quick enough, please add it into `Feign_Canal` group, and rename it to `Feign_Canal_ Pulsar`.

Comment (codelipenghui) [reply .github/workflows/plugins-test.yaml]: Sure

Comment (codelipenghui) [reply .github/workflows/plugins-test.yaml]: sure

File: test/plugin/scenarios/pulsar-scenario/config/expectedData.yaml [hunk 2]
```diff
@@ -0,0 +1,84 @@
+registryItems:
```
Comment (dmsolr) [inline test/plugin/scenarios/pulsar-scenario/config/expectedData.yaml]: Missing License Header here.

Comment (codelipenghui) [reply test/plugin/scenarios/pulsar-scenario/config/expectedData.yaml]: Thanks for reminding

**Comment by codelipenghui** [comment]
The Agent Plugin tests is still failed and then i check the test result, i got 404 there. How can i get some information about the failed tests.

**Comment by wu-sheng** [comment]
@codelipenghui Ignore that. We are moving to GitHub action. Jenkins jobs are removed, but because of some INFRA team issues, they are keeping showing up.

**Pull request merged**